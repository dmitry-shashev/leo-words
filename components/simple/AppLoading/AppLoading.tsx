import { FC } from 'react'

export const AppLoading: FC = () => {
  return (
    <div className="flex h-80">
      <div className="text-2xl text-blue-400 m-auto">Loading ...</div>
    </div>
  )
}
