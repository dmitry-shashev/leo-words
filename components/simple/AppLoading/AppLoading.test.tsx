import { expect, test } from 'vitest'
import { render } from '@testing-library/react'
import { AppLoading } from '@/components/simple/AppLoading/AppLoading'

test('AppLoading', () => {
  const { asFragment } = render(<AppLoading />)

  expect(asFragment()).toMatchSnapshot()
})
