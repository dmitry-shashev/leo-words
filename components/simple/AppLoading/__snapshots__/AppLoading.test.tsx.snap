// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppLoading 1`] = `
<DocumentFragment>
  <div
    class="flex h-80"
  >
    <div
      class="text-2xl text-blue-400 m-auto"
    >
      Loading ...
    </div>
  </div>
</DocumentFragment>
`;
